### PR TITLE
Grab mac addresses from computers with no NetworkAdapters field

### PR DESCRIPTION
**Pull request by ericz2002**

### CODE REVIEW

**Review by dkosteck — APPROVED** [review]
To me these changes look reasonable, I would wait for @adhil0 to review before merging, but I think you did a good job with this

**Review by adhil0 — APPROVED** [review]
Nice work!

**Pull request merged**
